hypothesis-python/tox.ini

```diff
@@ -71,11 +71,11 @@ deps =
 commands=
     nosetests tests/cover/test_testdecorators.py
 
-[testenv:pytest43]
+[testenv:pytest46]
 deps =
     -r../requirements/test.txt
 commands=
-    pip install pytest==4.3 pytest-xdist==1.25 pytest-forked==0.2
+    pip install pytest==4.6 pytest-xdist==1.34
```

Comment:
I bumped pytest-xdist to the latest version that supports pytest 4.x.

Not sure if pytest-forked still needs to be pinned; pytest-xdist 1.34 seems to work fine with the latest release.